Tests/ClientRuntimeTests/NetworkingTests/Streaming/StreamableHttpBodyTests.swift

```diff
@@ -12,60 +12,73 @@ import AwsCommonRuntimeKit
 
 final class StreamableHttpBodyTests: XCTestCase {
 
-    let testData = Data([0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A])
+    let testData = Data([
+        0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A,
+        0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A
+    ])
 
-    func testRead() throws {
-        let sut = StreamableHttpBody(body: .data(testData))
+    func test_streamWholeData() throws {
+        try read(data: Data(testData[..<11]))
+        try seek(data: Data(testData[11...]))
+    }
+
+    func test_streamSlicedData() throws {
+        try read(data: testData[..<11])
+        try seek(data: testData[11...])
```

Comment:
Creating a new `Data` by subscripting a larger `Data` with a range creates "sliced" `Data` where the buffer is shared between the old & new, but the `startIndex` for the new data may not be zero.